cmd/monaco/main.go

```diff
@@ -348,3 +361,144 @@ func getDownloadCommand(fs afero.Fs) cli.Command {
 	}
 	return command
 }
+
+func getDeployCommandV2(fs afero.Fs) cli.Command {
+	command := cli.Command{
+		Name:      "deploy",
+		Usage:     "deploys the given deployment manifesto",
+		UsageText: "deploy [command options] <path to deployment manifesto>",
+		ArgsUsage: "<path to deployment manifesto>",
+		Before: func(c *cli.Context) error {
+			err := util.SetupLogging(c.Bool("verbose"))
+
+			if err != nil {
+				return err
+			}
+
+			util.Log.Info("Dynatrace Monitoring as Code v" + version.MonitoringAsCode)
+
+			return nil
+		},
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:    "verbose",
+				Aliases: []string{"v"},
+			},
+			&cli.StringFlag{
+				Name:    "specific-environment",
```

Comment:
This would be a good opportunity to modify this: how about using `-e` or `-environment`. Not sure we need `specific`

Reply:
If you ask me, we should delete this file anyway. The new main is located 
in `cmd/monaco/v2/main.go`.

Reply:
I just kept this file as I wasn't sure how we want to go on with it.